<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TreeATE Dev</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_main"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuProject">
    <property name="title">
     <string>&amp;Project</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="action_Open"/>
    <addaction name="action_Save"/>
    <addaction name="separator"/>
    <addaction name="actionImport"/>
    <addaction name="action_Export"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionShrink_items_Ctrl"/>
    <addaction name="actionSpread_items_Ctrl"/>
    <addaction name="separator"/>
    <addaction name="actionWindows"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_Help"/>
    <addaction name="separator"/>
    <addaction name="action_About"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <widget class="QMenu" name="menu_Test_Units">
     <property name="title">
      <string>&amp;Test Items</string>
     </property>
     <addaction name="actionUpItem"/>
     <addaction name="actionDownItem"/>
     <addaction name="separator"/>
     <addaction name="actionInsert_Unit"/>
     <addaction name="actionInsert_Parameter"/>
     <addaction name="separator"/>
     <addaction name="actionRemove_Unit"/>
     <addaction name="actionRemove_Parameter"/>
     <addaction name="separator"/>
     <addaction name="actionAdd_Sub_Unit"/>
    </widget>
    <addaction name="action_Script"/>
    <addaction name="menu_Test_Units"/>
   </widget>
   <addaction name="menuProject"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>Toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_New"/>
   <addaction name="action_Open"/>
   <addaction name="action_Save"/>
   <addaction name="separator"/>
   <addaction name="actionImport"/>
   <addaction name="action_Export"/>
   <addaction name="actionUpItem"/>
   <addaction name="actionDownItem"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget_model">
   <property name="windowTitle">
    <string>Project files</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_model">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_model"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="action_New">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new.png</normaloff>:/new.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/open.png</normaloff>:/open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/save.png</normaloff>:/save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About...</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/input.png</normaloff>:/input.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Import...</string>
   </property>
  </action>
  <action name="action_Export">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/output.png</normaloff>:/output.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Export...</string>
   </property>
  </action>
  <action name="actionShrink_items_Ctrl">
   <property name="icon">
    <iconset resource="../TreeATE/res.qrc">
     <normaloff>:/minus.png</normaloff>:/minus.png</iconset>
   </property>
   <property name="text">
    <string>Shrink items</string>
   </property>
   <property name="toolTip">
    <string>Shrink items</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionSpread_items_Ctrl">
   <property name="icon">
    <iconset resource="../TreeATE/res.qrc">
     <normaloff>:/add.png</normaloff>:/add.png</iconset>
   </property>
   <property name="text">
    <string>Spread items</string>
   </property>
   <property name="toolTip">
    <string>Spread items</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+=</string>
   </property>
  </action>
  <action name="actionWindows">
   <property name="text">
    <string>Windows</string>
   </property>
  </action>
  <action name="actionInsert_Unit">
   <property name="text">
    <string>Insert Item</string>
   </property>
  </action>
  <action name="actionInsert_Parameter">
   <property name="text">
    <string>Insert Parameter</string>
   </property>
  </action>
  <action name="actionRemove_Unit">
   <property name="text">
    <string>Remove Item</string>
   </property>
  </action>
  <action name="actionRemove_Parameter">
   <property name="text">
    <string>Remove Parameter</string>
   </property>
  </action>
  <action name="actionAdd_Sub_Unit">
   <property name="text">
    <string>Add Sub Item</string>
   </property>
  </action>
  <action name="action_Script">
   <property name="text">
    <string>&amp;Script</string>
   </property>
  </action>
  <action name="actionUpItem">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/up.png</normaloff>:/up.png</iconset>
   </property>
   <property name="text">
    <string>Up Item</string>
   </property>
  </action>
  <action name="actionDownItem">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/down.png</normaloff>:/down.png</iconset>
   </property>
   <property name="text">
    <string>Down Item</string>
   </property>
  </action>
  <action name="action_Help">
   <property name="text">
    <string>&amp;Help...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../TreeATE/res.qrc"/>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
